apache/superset
# test: fix table chart sort order E2E tests

### SUMMARY

Follow up of #12930 .  Fix the table chart E2E test currently failing in master. The test expects table chart to be sorted by first metric in descending order when a "Sort by" metric is not set, regardless of if "Sort descending" is selected. This is the original logic pre table chart API migration (#10270).

Needs to wait for https://github.com/apache-superset/superset-ui/pull/935


### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


#### After

Correctly sorted by first metric in descending order:

![Snip20210203_1](https://user-images.githubusercontent.com/335541/106857379-d6263600-6674-11eb-8230-73c93a5a869d.png)

#### Before

Incorrectly sort by first metric in ascending order.

### TEST PLAN

CI and

1. Go to table chart
2. Select a metric, don't select sort by metric, **don't check "sort descending"**
3. Your chart should sort by first metric in **descending** order

### ADDITIONAL INFORMATION
